### ISSUE TITLE
cmake procedure missing in WW3 wiki

### ISSUE THREAD

**Issue by mickaelaccensi** [NOAA-EMC/WW3#1067]
I'm not able to find any information about WW3 installation based on cmake.
It should be in the quick start guide at least

**Comment by aliabdolali** [comment]
It is under model/readme.md
Updating two places would be difficult in my opinion.

https://github.com/NOAA-EMC/WW3/blob/develop/model/README.md

**Comment by JessicaMeixner-NOAA** [comment]
@mickaelaccensi I agree the wiki should be consistent with the procedure expected in the code and right now it is not.   While updating in multiple places is difficult, any instructions need to be removed for the wiki.  @MatthewMasarik-NOAA and I are working on updating the wiki although it's a lower priority item for us both so the progress is slow.  Moreover it's on my plate to make sure our documentation is up to date and remove the gnu build once that's complete from the code. Again it's a low priority issue so it's taking much longer than anticipated.

**Comment by mickaelaccensi** [comment]
ok thanks.So, the best would probably be to redirect to the readme instead of duplicating the installation procedure on the wiki to avoid outdated information.
